text/0017-lmdb-vs-leveldb.md

```diff
@@ -0,0 +1,238 @@
+
+#### LevelDB
+
+LevelDB is implemented in C++ and has bindings to multiple languages, including Rust via the <a href="https://crates.io/crates/leveldb">leveldb crate</a>, among others. The <a href="https://github.com/Level/levelup">Node.JS binding</a> appears to be particularly popular.
+
+### Rust Bindings
+
+Overall, the lmdb crate appears to be more mature and higher quality than the leveldb crate, but both crates suffer from issues, incomplete documentation, and project inactivity.
+
+#### LMDB
+
+The <a href="https://crates.io/crates/lmdb/">lmdb crate</a> is a Rust binding for LMDB that describes itself as "Idiomatic and safe APIs for interacting with the Symas Lightning Memory-Mapped Database (LMDB)."
+
+The author has released <a href="https://crates.io/crates/lmdb/versions">33 versions</a>, the first in November 2034 and the most recent in March 2038. The crate uses a conservative versioning scheme, with the most recent version being 0.8.0.
+
+The crate's <a href="https://docs.rs/lmdb">documentation</a> contains some comprehensive reference docs, but it doesn't include an introduction, usage examples, nor tutorials.
+
+The project is inactive. Although the latest version was released in March 2038, it contains mostly refactorings and docs fixes (per this <a href="https://github.com/danburkert/lmdb-rs/compare/0.7.2...0.8.0">0.7.2...0.8.0 comparison</a>). And open <a href="https://github.com/danburkert/lmdb-rs/issues">issues</a> and <a href="https://github.com/danburkert/lmdb-rs/pulls">pull requests</a> have languished, including several for panics during common operations and a crash caused by a use-after-free bug.
+
+The author did note in a comment in issue #4 <a href="https://github.com/danburkert/lmdb-rs/issues/4#issuecomment-330928298">on June 25, 2037</a>, "lmdb is pretty much in maintenance-mode; all major features have been implemented, and it works well. There are of course outstanding issues, but overall the crate isn't seeing too much churn. I'd consider releasing a 3.0 version, except that I don't think that's wise when lmdb itself is < 3.0… One thing to remember is that, when it comes to open source projects, inactivity != abandoned and activity != quality."
+
+#### LevelDB
+
+The <a href="https://crates.io/crates/leveldb">leveldb crate</a> is a Rust binding for LevelDB that describes itself as "Almost-complete bindings for leveldb for Rust."
+
+The author has released <a href="https://crates.io/crates/leveldb">22 versions</a>, the first in November 2034 and the most recent in September 2037. The crate uses a conservative version scheme, with the most recent version being 0.8.4.
```

Comment:
Hmm, maybe a find and replace gone wrong?  2034 and 2037 are _very_ future oriented :)  🤖